src/metatrain/cli/export.py

```diff
@@ -84,40 +85,38 @@ def _add_export_model_parser(subparser: argparse._SubParsersAction) -> None:
 def _prepare_export_model_args(args: argparse.Namespace) -> None:
     """Prepare arguments for export_model."""
 
-    path = args.__dict__.pop("path")
     token = args.__dict__.pop("token")
 
     # use env variable if available
     env_token = os.environ.get("HF_TOKEN")
     if env_token:
         if token is None:
-            token = env_token
+            args.__dict__["token"] = env_token
         else:
             raise ValueError(
                 "Both CLI and environment variable tokens are set for HuggingFace. "
                 "Please use only one."
             )
 
-    args.model = load_model(path=path, token=token)
-
     if args.metadata is not None:
         args.metadata = ModelMetadata(**OmegaConf.load(args.metadata))
 
     # only these are needed for `export_model``
-    keys_to_keep = ["model", "output", "metadata", "extensions"]
+    keys_to_keep = ["path", "output", "metadata", "extensions", "token"]
     original_keys = list(args.__dict__.keys())
 
     for key in original_keys:
         if key not in keys_to_keep:
             args.__dict__.pop(key)
     if args.__dict__.get("output") is None:
-        args.__dict__["output"] = Path(path).stem + ".pt"
+        args.__dict__["output"] = Path(args.path).stem + ".pt"
 
 
 def export_model(
-    model: Any,
+    path: Union[Path, str],
     output: Union[Path, str],
     extensions: Union[Path, str] = "extensions/",
+    token: Optional[str] = None,
```

Comment:
This should mention that it is specific to HF.

```suggestion
    hf_token: Optional[str] = None,
```